internal/controller/logicalreplication_controller.go

```diff
@@ -133,25 +133,25 @@ func (i *LogicalReplicationIteration) readCredentails() error {
 	return nil
 }
 
-func (i *LogicalReplicationIteration) connectDBs() error {
-	publishingDb, err := replication.DBConnect(i.pubCreds)
+func (i *LogicalReplicationIteration) connectDB(creds replication.DatabaseCredentials) (*sql.DB, error) {
+	db, err := replication.DBConnect(creds)
 	if err != nil {
-		i.log.Error(err, "connecting to publication db")
-		return NewReplicationError(ConnectError, err)
+		i.log.Error(err, fmt.Sprintf("connecting to %s db", creds.DatabaseName))
+		return nil, NewReplicationError(ConnectError, err)
 	}
-	i.pubDB = publishingDb
-
-	i.log.Info("connected to publishing database")
+	i.log.Info(fmt.Sprintf("connected to %s database", creds.DatabaseName))
```

Comment:
Debug log maybe?

Reply:
This logger has just Info() or Error().

Reply:
Hmm. It might be too verbose to leave this here.